libwidgets/Chrome/BasicBreadcrumbsEntry.vala

```diff
@@ -40,10 +40,10 @@ namespace Marlin.View.Chrome {
         }
 
         public bool hide_breadcrumbs { get; set; default = false; }
-        public const double MINIMUM_LOCATION_BAR_ENTRY_WIDTH = 36;
+        public const double MINIMUM_LOCATION_BAR_ENTRY_WIDTH = 10;
         public const double MINIMUM_BREADCRUMB_WIDTH = 12;
         public const double COMPLETION_ALPHA = 0.5;
-        public const int ICON_WIDTH = 48;
+        public const int ICON_WIDTH = 6;
```

Comment:
I am not sure the ICON_WIDTH constant is needed (any more) but while it is there it would be better to set it to a valid IconSize - the smallest available is 16px.  You can reduce the minimum entry width to keep the total to say 24px.  The breadcrumbs entry is in need of refactoring anyway and I'll look into whether there are any regressions if ICON_WIDTH is removed.

Reply:
After setting ICON_WIDTH to 0, I noticed that it needs to be set to a proper value to ensure that this doesn't happen while resizing the window :
![screenshot from 2019-02-26 19 33 59](https://user-images.githubusercontent.com/1344696/53438229-bead8f80-39ff-11e9-9910-c691592d636e.png)
After fiddling around with both values, breadcrumbs bar behaves like this on last commit and access to path on the tightest resize is maintained. I may have overlooked this on previous commit...
![screenshot from 2019-02-26 19 38 32](https://user-images.githubusercontent.com/1344696/53438358-06341b80-3a00-11e9-9a4e-384e32d2c643.png)
![screenshot from 2019-02-26 19 38 44](https://user-images.githubusercontent.com/1344696/53438369-09c7a280-3a00-11e9-9ade-1b6d4a4e5c9a.png)